packages/fxa-react/configs/tailwind.js

```diff
@@ -63,24 +49,15 @@ module.exports = {
         100: '25rem',
       },
       inset: {
-        '1/2': '50%',
-        '-px': '-1px',
-        '-3': '-0.75rem',
-        '-50': '-11.50rem',
-        '-52': '-13rem',
-        0.5: '0.125rem',
-        3: '0.75rem',
-        6: '1.25rem',
```

Comment:
1.25rem corresponds to TW's default class inset-5, so I removed the custom configuration and replaced inset-6 with inset-5 where it is used in BentoMenu.